.github/workflows/test_cloudpods.yml

```diff
@@ -0,0 +1,123 @@
+name: Test Released Cloud Pods
+
+on:
+  schedule:
+    # “At 00:00 on Saturday.”
+    - cron: "0 0 * * 6"
+  workflow_dispatch:
+  push: # TODO remove just for testing
+    paths-ignore:
+      - 'README.md'
+    branches:
+      - cloudpod_workflow
+permissions:
+  contents: write
+
+jobs:
+  get-releases:
+    name: Retrieve Released Cloud Pods
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - id: set-matrix
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        output=$(gh api repos/$GITHUB_REPOSITORY/releases |  jq -r '.[] | select(.tag_name|startswith("v")|not) | .tag_name'')
```

Comment:
if we remove the older releases in this repo, using `jq  '[.[].tag_name]'` should be sufficient. the older releases do not follow the release pattern, where we use the tag_name as well for the LocalStack version tag, when we run the tests